Add blog post announcing the upcoming OMERO.web Django upgrade work

sbesson opened the pull request:
Staged for review at https://snoopycrimecop.github.io/www.openmicroscopy.org/2022/02/17/omeroweb-django-upgrade.html

snoopycrimecop commented:
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

snoopycrimecop commented:
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

will-moore commented on _posts/2022-02-17-omeroweb-django-upgrade.md:
```diff
@@ -0,0 +1,68 @@
+
+Running a version of any software that the authors consider unsupported is
+risky; if any bugs of either class are found they will not be fixed upstream.
+However, we are not currently aware of any security or data loss bugs in th
```
`th`

will-moore commented on _posts/2022-02-17-omeroweb-django-upgrade.md:
```diff
@@ -0,0 +1,68 @@
+
+For the future safety of the community as well as enhanced developer
+ergonomics we need to upgrade Django.  This was first attempted, targetting
+Django 2.2, during the summer of 2020 [^2].  Unfortunately, that work neve
```
`neve`

will-moore commented on _posts/2022-02-17-omeroweb-django-upgrade.md:
```diff
@@ -0,0 +1,68 @@
+
+Plugin developers are encouraged to review the release notes, in particular
+the *Backwards incompatible* and *Features deprecated* for all major Django
+versions since 1.11.x to ensure they can be complaint with the new APIs.  They
```
complaint  -> compliant

will-moore commented on _posts/2022-02-17-omeroweb-django-upgrade.md:
```diff
@@ -0,0 +1,68 @@
+The OME team will be publishing a migration guide along with the release
+candidate to draw the attention of plugin developers to specific OMERO.web
+idioms and design patterns that __will__ change in this new major version.
+For those developers wishing to observe this process in real time may do so by
```
Remove "For ". "Those..." reads better.

will-moore reviewed (commented):
I wonder if we should include something about our plan to udgrade the omero-web plugins?
From initially looking at iviewer and figure, it seems that the number of changes required for the upgrade are quite minimal and we should probably release the plugins along with omero-web.

sbesson commented:
Updated the date of the post and addressed all the typos. From yesterday's discussion, our intent is very much to be able to release as much as a full ecosystem (omero-web + omero-apps). This blog post was written before we had evidence about the feasibility of the apps upgrade so it omits a roadmap for the Web apps. 

Unless someone wants to propose an amendment, my inclination would be to get this in. Possible by the end of February, we will have a better idea of the impact and the release schedule and we can follow-up with another post about the apps and the release timeline?
